### PR TITLE
Add initial code imported from the SDK

### PR DESCRIPTION
**Pull request by llucax**
- **Import implementation from the SDK v1.0.0rc601**
- **Add an introduction and example to the package documentation**
- **Include the package documentation in the website home**
- **Add a link to the documentation in the README**

### CODE REVIEW

File: src/frequenz/quantities/__init__.py [hunk 1]
```diff
@@ -1,7 +1,80 @@
 # License: MIT
 # Copyright © 2024 Frequenz Energy-as-a-Service GmbH
 
-"""Types for holding quantities with units."""
+"""Types for holding quantities with units.
+
+This library provide types for holding quantities with units. The main goal is to avoid
```
Comment (Marenz) [inline src/frequenz/quantities/__init__.py:6]: Maybe remove the first sentence as it is equal to the headline

**Review by Marenz — DISMISSED** [review]

File: src/frequenz/quantities/__init__.py (hunk 1 above)
Comment (llucax) [reply src/frequenz/quantities/__init__.py:6]: This is a title/summary, like with git commits. There are some options or places where only the summary line is shown.

**Comment by llucax** [comment]
I updated the branch to v1.x.x to follow the suggestion in https://github.com/frequenz-floss/frequenz-sdk-python/issues/852. I think this is mature enough to go with a v1.0.0. If we want to make breaking changes we can start a new v2.x.x branch, but it will make us asses the cost-benefit from doing that and avoid cheap breaking changes...

**Comment by llucax** [comment]
Added release notes assuming v1.0.0 will be released after this is merged.

**Review by Marenz — APPROVED** [review]

**Pull request merged**